test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -14,9 +14,38 @@ namespace Microsoft.DotNet.Docker.Tests
     public class ImageTests
     {
         private static string ArchFilter => Environment.GetEnvironmentVariable("IMAGE_ARCH_FILTER");
-        private static string OsFilter { get; set; } = Environment.GetEnvironmentVariable("IMAGE_OS_FILTER");
+        private static string OsFilter => Environment.GetEnvironmentVariable("IMAGE_OS_FILTER");
         private static string VersionFilter => Environment.GetEnvironmentVariable("IMAGE_VERSION_FILTER");
 
+        private static ImageDescriptor[] LinuxTestData = new ImageDescriptor[]
+            {
+                new ImageDescriptor { DotNetCoreVersion = "1.0", SdkVersion = "1.1" },
+                new ImageDescriptor { DotNetCoreVersion = "1.1", RuntimeDepsVersion = "1.0" },
+                new ImageDescriptor { DotNetCoreVersion = "2.0" },
+                new ImageDescriptor { DotNetCoreVersion = "2.0", OsVariant = OS.Jessie },
+                new ImageDescriptor { DotNetCoreVersion = "2.0", OsVariant = OS.Stretch, SdkOsVariant = "", Architecture = "arm" },
```

Comment:
Nit - please check line length.

Reply:
This is the max length showable in the github viewer without clipping.  Leaving as is.